src/statemanager/WfsPermalink.js

```diff
@@ -42,6 +42,8 @@ import olFormatWFS from 'ol/format/WFS.js';
  * @property {number} [totalFeatureCount] If `tooManyResults` is `true`, this contains the total number of
  *    features.
  * @property {string} [identifierAttributeField] The field to use to identify the feature.
+ * @property {string[]} [mergeComposants] The original types merged in merged result.
+ * @property {string[]} [requestPartners] The types of the request (which are summed in totalFeatureCount).
```

Comment:
Could you please search `types` in this PR and looks if you can use a better name. `Type` is not clear to me.
If this name is used somewhere else, please keep it for coherence.